Gemfile

```diff
@@ -28,6 +28,8 @@ group :development, :test do
   gem "pry-rails"
   gem "pry-nav"
   gem "faker"
+  gem "factory_girl_rails"
```

Comment:
@andela-echigbo added this line too, maybe you should take a look at his branch so as to avoid a merge conflict.